posnums.c

```diff
@@ -0,0 +1,67 @@
+/*MADELEINE CHARITY Write a program posnums.c that reads 2 positive integers from the user, and prints the average of the two (this should be printed as a floating point number.), whether each number is a prime number or not, and whether the two numbers are relatively prime or not (the only positive integer that divides both of them evenly is 1).*/
+#include <stdio.h>
+int main()
+{
+	int x;
+	int y;
+	float avg;
+	int i=1;
+	
+	do{
+		printf("Enter a number:\n");
+		if (scanf("%d", &x)==0)
+		{ 
+			printf("Not valid\n");
+			while(getchar()!='\n');
+		}
+		else 
+			break;
+	}
+	while (x==0);
+	do {
+		printf("Enter another number:\n");
+		if (scanf("%d",&y)==0)
+		{
+			printf("Not valid\n");
+			while(getchar()!='\n');
+		}
+		else 
+			break;
+	}
+	while (y==0);
+//	printf("test");
+	avg=(((float)x+(float)y)/2);
+	printf("The average is %f\n", avg);
+	printf("test");
+	while (i<x);
+	{
+		i++;
+		if (x%i==0)
+		{
+			printf("%d is not prime\n", x); 
+		}
+		else if (i<x)
+			continue;
```

Comment:
this is where your error came from - continues can only occur in for and while loops. if you deleted this, it would compile.